api/v1/model_handlers.go

```diff
@@ -272,6 +274,20 @@ func (a *Api) RegisterModel(container *restful.Container) {
 		Param(ws.QueryParameter("end", "End time for requested metric").DataType("string")).
 		Writes(MetricResult{}))
 
+	// The /sum/namespaces/{namespace-name}/pods/{pod-name}/metrics/{metric-name} endpoint exposes
+	// an aggregated metric for a list od pods of the model.
+	ws.Route(ws.GET("/sum/namespaces/{namespace-name}/pods/{pod-list}/metrics/{metric-name}").
```

Comment:
The existing API structure is meant to resemble the Kubernetes API, by having subpaths that indicate increased specificity in the cluster structure. 

Because of that, something like /namespaces/{namespace-name}/pods/{pod-list}/metrics/{metric-name} would be cleaner. To implement that, we could change the existing handler for that path to attempt to extract a comma-separated list of podnames.